models/istio_config.go

```diff
@@ -36,6 +37,7 @@ type IstioConfigDetails struct {
 	QuotaSpec        *QuotaSpec          `json:"quotaSpec"`
 	QuotaSpecBinding *QuotaSpecBinding   `json:"quotaSpecBinding"`
 	Policy           *Policy             `json:"policy"`
+	MeshPolicy       *Policy             `json:"meshPolicy"`
```

Comment:
if MeshPolicy is implemented with a different Kind in the backend, I'd add a new type here.
I know redundant, but it can help to maintain one-to-one mapping.
I understand the optimization, but perhaps doing this redundant type will help.